### ISSUE TITLE
BR1 - Banner reads the alt text for the next image in the slide show

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#196]
I ended up with an odd issue while using the screen reader in Chrome where it would read the alt text for the next image. I produced the following from NVDA by moving the banner on [OAC's stage site](https://www-stage.uoguelph.ca/oac/) to the Hazelnut slide and then going down through the banner:
```
banner landmark 
button
 Pause
graphic
Close crop ofthree Geneva nuts (hazelnuts) on black cloth
graphic
Ariel image of people walking on a green grass pathway and trees on a city street
link
Hazelnuts, An Opportunity for Ontario
```
It's reading the alt text for the appropriate image, followed by the alt text for the next slide, and then it reads the linked title. Firefox seems like it's picking it up the way it should.

**Comment by mmafe** [comment]
@jpinsent Is this still occurring?

**Comment by jpinsent** [comment]
@mmafe This issue still seems to be occurring. I can recreate it when I either navigate to or let the banner autoplay to the hazelnut slide and then pause the slideshow. The screen reader will pick up the image for the next slide in addition to the current one.

**Comment by jpinsent** [comment]
It seems like it's working now that the changes have been pushed to the stage site. Only the appropriate image is read, and it is now read with its corresponding slide number for additional context.

This issue has been resolved.
